react-webapp/README.md

```diff
@@ -0,0 +1,94 @@
+## OMERO webapp example
+
+This project is an OMERO.web example app created with [Create React App](https://github.com/facebook/create-react-app).
+
+You can run this project in development mode or as an OMERO.web Django app.
+
+The notes below are largely from `Create React App` but have been updated with
+changes made to connect to OMERO in dev or production modes.
+
+To get started:
+
+    $ cd react-webapp
+    $ npm install
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.<br>
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br>
+You will also see any lint errors in the console.
+
+The app will try to connect to an OMERO.web server at http://localhost:4080
+using an existing session.<br>
+You will need to have CORS enabled and
+to be logged-in to http://localhost:4080/webclient.<br>
+To use a different server, edit `dev_omeroweb_index` in `public/index.html`.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder and copies the
+html and static files to the Django app in `react_webapp`.<br>
+
+You also need to install the app into your `omero-web` environment:
+
+    $ cd react-webapp
+    $ pip install -e .
+
+You will need to have the app configured in your OMERO.web install:
+
+    $ omero config append omero.web.apps '"react_webapp"'
+
+The app will be run as an OMERO.web app at e.g. http://localhost:4080/react_webapp/.<br>
+
+It bundles React in production mode and optimizes the build for the best performance.
+
+See the Create React App section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.<br>
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+
+If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+
+You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
```

Comment:
don't -> do not
 shouldn’t -> should not
couldn’t -> could not

Reply:
These are all from Create React App which generated the initial README, not my words.